src/Security/Authorization/Core/src/Properties/AssemblyInfo.cs

```diff
@@ -1,9 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.Runtime.CompilerServices;
-using Microsoft.AspNetCore.Authorization;
-
-// Microsoft.AspNetCore.Http.Abstractions
-[assembly: TypeForwardedTo(typeof(IAuthorizeData))]
-[assembly: TypeForwardedTo(typeof(IAllowAnonymous))]
```

Comment:
Why remove these? We should still type-forward these so it's not a breaking change.